dev/TabView/TabView_themeresources.xaml

```diff
@@ -6,11 +6,11 @@
 
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Light">
-            <StaticResource x:Key="TabViewBackground"                        ResourceKey="SystemChromeLowColor" />
+            <StaticResource x:Key="TabViewBackground"                        ResourceKey="SystemControlBackgroundBaseLowBrush" />
```

Comment:
These theme resource changes aren't called out anywhere I can see in the PR description - are they intended?

Reply:
I forgot to mention them! Added it to the PR description. Yes, they're intentional; we had a big discussion about colors and this is where we landed for now.

Reply:
Cool, thanks. :)